src/steamship/agents/schema/agent.py

```diff
@@ -57,3 +59,11 @@ def messages_to_prompt_history(self, messages: List[Block]) -> str:
             elif role == RoleTag.AGENT:
                 as_strings.append(f"Agent: {block.text}")
         return "\n".join(as_strings)
+
+    @staticmethod
+    def search_to_search_history(search_results: SearchResults) -> str:
+        blocks = []
```

Comment:
Could SearchResults override __iter__ to encapsulate this behavior? 

If yes, we would not need the search_to_search_history wrapper. Instead we could use messages_to_prompt_history directly.

Reply:
I did this slightly differently, by moving the content of the method to SearchResults, but I think still achieving the desired effect.  I didn't make it the iter because iter on SearchResults would mean SearchResult, not Block, to me.